cluster/manifests/fabric-gateway/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-{{ $version := "master-200" }}
+{{ $version := "master-208" }}
```

Comment:
I think both 208 and 209 are ok, but would like it to be 209.

Reply:
yes because of e2e weirdness I agree